test/layout.spec.ts

```diff
@@ -302,16 +302,21 @@ describe('Layout matchers', function () {
     describe('Detect sequence of elements', function () {
```

Comment:
Roger that. And [this](https://github.com/wixplosives/core3-containers/pull/34) pull-request, once merged, should fix the failing liabilities.